grails-app/views/phylo/show.gsp

```diff
@@ -285,7 +299,8 @@
             defaultValue: 'taxon_name'
         },
         downloadReasonsUrl: config.downloadReasonsUrl,
-        spUrl: config.spUrl
+        spUrl: config.spUrl,
+        records: records
```

Comment:
it is not in character. it is in map. records is initialised before map so instead of records getting map ref, map gets records ref.